gui/packages/ubuntupro/lib/pages/widgets/page_widgets.dart

```diff
@@ -91,17 +91,20 @@ class ColumnLandingPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final lang = AppLocalizations.of(context);
+    final theme = Theme.of(context);
+    final dark = theme.brightness == Brightness.dark;
 
     return Pro4WSLPage(
       body: Stack(
         fit: StackFit.expand,
         children: [
-          Positioned.fill(
-            child: Image.asset(
-              'assets/05_suru2_dark_2K.jpg',
-              fit: BoxFit.fill,
+          if (dark)
```

Comment:
If you want to get 100% coverage in your PR you can easily add this line to one of the Landscape and Subscribe pages:

```dart
tester.view.platformDispatcher.platformBrightnessTestValue =
        Brightness.dark;
```

Reply:
This shouldn't be relevant now, since we're removing the background image entirely.